insights/specs_default.py

```diff
@@ -357,6 +357,7 @@ def httpd_cmd(broker):
     oc_get_pvc = simple_command("/usr/bin/oc get pvc -o yaml --all-namespaces", context=OpenShiftContext)
     crt = simple_command("/usr/bin/find /etc/origin/node /etc/origin/master -type f -path '*.crt'")
     openshift_certificates = foreach_execute(crt, "/usr/bin/openssl x509 -noout -enddate -in %s")
+    openvswitch_other_config = simple_command("/usr/bin/ovs-vsctl -t 5 get Open_vSwitch . other_config")
```

Comment:
We might need to add a `simple_file` for this new simple_command to 'insights/specs_insights_archive.py'. I have asked Chris Sams if this can be done automatically. Let's just keep this PR here un-merged and do it after got feedback from Chris.

Reply:
@chenlizhong , please use the 'mangle' command to generate a "simple_file" for this simple_command and add it into 'insights/specs_insights_archive.py'

Reply:
@csams can you respond to this question?

Reply:
Hi @bfahr - Chris has answered my question in the email with a subject "Changes to Specs in Core 3.x" and @chenlizhong has added the corresponding 'simple_file' to 'insights/specs_insights_archive.py'. I'll go ahead and merge this PR at first. I there are any other items should be updated for the new 3.x style, I think we can raise another PR for it.